### ISSUE TITLE
Add homepage content to mirror production site

### ISSUE THREAD

**Issue by NWHarvest** [openseattle/northwest-harvest-growing-connections#26]

**Comment by alimon808** [comment]
Restore language?  Is this a reference to the homepage content?

**Comment by NWHarvest** [comment]
Yes. Restore may not have been the correct language. But the test site currently doesn't include the language from the live site.

**Comment by alimon808** [comment]
@NWHarvest homepage content is stored/retrieved from db.  Please export DisplayMessages & DisplayDescriptions tables.
